Gruntfile.js

```diff
@@ -256,7 +256,11 @@ module.exports = function (grunt) {
   });
 
   grunt.registerTask('default', ['newer:babel', 'less', 'newer:copy:dev', 'shell:electron', 'watchChokidar']);
-  grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress']);
+  if (!IS_WINDOWS) {
+    grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress']);
+  }else {
+    grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'copy:windows', 'rcedit:exes', 'compress']);
```

Comment:
No, it's not. My first idea was to avoid the execution of the steps that was not possible to execute on Windows. But I think you're right. So, I'm doing a change to remove copy:osx and to change from electron to electron:windows. You should see now the latest version of the commit.